src/components/atoms/Description/Description.tsx

```diff
@@ -0,0 +1,9 @@
+import styles from './Description.module.scss';
+
+export interface Props {
+  children: React.ReactNode;
```

Comment:
Please, also add an import of `React` so `React.ReactNode` type is correctly interpreted in IDEs